includes/Dashboard.php

```diff
@@ -123,6 +123,7 @@ public function load_stories_dashboard() {
 		// TODO Preload templates.
 		$preload_paths = [
 			'/web-stories/v1/fonts',
+			'/wp/v2/settings',
```

Comment:
Why preload something that might not even be accessed? And is behind a feature flag anyway.

Reply:
What does it hurt to preload these?

Reply:
Page load time will be (minimally) increased by this, no?

Reply:
I did some testing, it was around 5-15ms. Settings is a pretty little weight REST API request, as all the options are preloaded already. Where as a request to the settings endpoint is around 55ms, as there is a DNS / transport overhead to any http request.